numpy/core/fromnumeric.py

```diff
@@ -3700,8 +3709,15 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
     where : array_like of bool, optional
         Elements to include in the variance. See `~numpy.ufunc.reduce` for
         details.
-
+        
```

Comment:
```suggestion

```

Reply:
I hope I fixed it.

Reply:
.. versionadded:: 1.20.0

Appears with two white lines around it. I copied that.